engine/classes/Elgg/Database/Plugins.php

```diff
@@ -410,11 +402,10 @@ function find($status = 'active') {
 			'type' => 'object',
 			'subtype' => 'plugin',
 			'limit' => ELGG_ENTITIES_NO_VALUE,
-			'selects' => array('plugin_oe.*', 'ps.value'),
```

Comment:
will investigate further... seems to be working right now... but do not know why exactly :)